.github/workflows/test-deploy.yml

```
@@ -104,7 +104,12 @@ jobs:
         with:
           registry-url: https://registry.npmjs.org
       - run: npm install
-      - run: npm ${{ inputs.release && 'publish' || 'publish --dry-run' }}
+      - run: |
+          if [$GITHUB_RUN_ATTEMPT == 1 ]; then
+            npm ${{ inputs.release && 'publish' || 'publish --dry-run' }}
+          else
+            echo "Job being re-run, skipping"
+          fi
```

:(Sounds of angry gophers sharpening their teeth)_

```suggestion
      - if: github.run_attempt == 1
        run: npm ${{ inputs.release && 'publish' || 'publish --dry-run' }}
```